platform-workflow-module/src/main/java/com/softicar/platform/workflow/module/workflow/task/delegation/AGWorkflowTaskDelegation.java

```diff
@@ -1,9 +1,19 @@
 package com.softicar.platform.workflow.module.workflow.task.delegation;
 
+import com.softicar.platform.common.date.DayTime;
 import com.softicar.platform.emf.trait.IEmfTrait;
 import com.softicar.platform.workflow.module.workflow.task.AGWorkflowTask;
+import java.util.Optional;
 
 public class AGWorkflowTaskDelegation extends AGWorkflowTaskDelegationGenerated implements IEmfTrait<AGWorkflowTaskDelegation, AGWorkflowTask> {
 
-	// nothing to add
+	public Optional<DayTime> getLastModificationTimeFormLog() {
```

Comment:
Performance of this query might degrade. We could instead add a column like this to `AGWorkflowTaskDelegation.sqml`:
`	AGTransaction transaction`
See `AGDemoInvoice.sqml` for an example. That column gets automagically updated with the record, always referring to the most recent transaction. This way, we could spare using the logs here.

Reply:
added ToDo as discussed